qa/suites/fs/upgrade/upgraded_client/tasks/1-client-upgrade.yaml

```diff
@@ -1,6 +1,7 @@
 tasks:
 - install.upgrade:
     client.0:
+- ceph.healthy:
```

Comment:
nit: maybe this should be after the `print` below?

Reply:
Fixed.